### PR TITLE
Adding lighting talks

### PR DESCRIPTION
**Pull request by AyhamAlzoubi**

### CODE REVIEW

File: source/_posts/2015-04-26-lighting-talks-1.markdown [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+https://www.wireshark.org/
+
+
+* [Hossam](http://tech.namshi.com/team/#Hossam%20Faris), showed us some good tips and advice on how to use promises:
+
+http://taoofcode.net/promise-anti-patterns/
+
+
+* [Cirpo](http://tech.namshi.com/team/#Alessandro%20Cinelli), shared how tmux and CDPATH can enhance your terminal usage experience:
+
```
Comment (odino) [inline source/_posts/2015-04-26-lighting-talks-1.markdown]: should also mention that the demo was pretty hilarious :)

Comment (AyhamAlzoubi) [inline source/_posts/2015-04-26-lighting-talks-1.markdown]: :)

**Comment by odino** [comment]
BTW shall we merge this?

**Comment by AyhamAlzoubi** [comment]
merging

**Pull request merged**